packages/liquidation-sdk-viem/src/LiquidationEncoder.ts

```diff
@@ -588,4 +599,228 @@ export class LiquidationEncoder<
 
     return { dstAmount };
   }
+
+  // MIDAS
+
+  async handleMidasTokens(collateralToken: Address, seizedAssets: bigint) {
+    if (!Midas.isMidasToken(collateralToken, this.client.chain.id)) {
```

Comment:
This check is also being performed in `whitelisted-erc4626-1inch.ts`; do you want it in both places?